dashboard/app/views/reports/search_for_teachers.html.haml

```diff
@@ -0,0 +1,65 @@
+- content_for(:head) do
+  = stylesheet_link_tag    'jquery.handsontable.full', media: 'all'
+  = javascript_include_tag 'jquery.handsontable.full'
+%h1 Search for Teachers
+
+%table
+  %tr
+    %td
+      Email Filter
+    %td
+      %input#emailFilter
+  %tr
+    %td
+      Address Filter
+    %td
+      %input#addressFilter
+
+%br/
+
+%center
+  %a#search
+    %button Search
+
+%br/
+
+#teacherTable
+
+:javascript
```

Comment:
Because I was blindly copying the "template" in level_completions.html.haml.

Let me convert to an HTML table, a la https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/reports/usage.html.haml.